src/ralph/static/src/js/fill-fields.js

```diff
@@ -0,0 +1,76 @@
+// Fill fields in bulk edit and split asset forms
+'use strict';
+
+$(document).ready(function () {
+    // We use absolute positioning for toolbar therefore body must be relative
+    $('body').css('position', 'relative');
+    // Disable autocomplete without cluttering html attributes
+    $('input').attr('autocomplete', 'off');
+
+    $('.result-list input[type=text]:not(.no-fillable), .result-list select').blur(function () {
+        $('#float_toolbar').hide();
+    });
+
+    var toggle_toolbar = function (input) {
+        var toolbar = $('#float_toolbar');
+        var offset = $(input).offset();
+        var width = input.clientWidth;
+        var distance_left = -30;
+        var distance_top = -134;
+
+        /* Clear data */
+        $(toolbar).data('input_name', '');
+        $(toolbar).data('input_id', '');
+
+        /* For fields without id set, save its name as selector */
+        if (input.id === '') {
+            $(toolbar).data('input_name', input.name);
+        } else {
+            $(toolbar).data('input_id', input.id);
+        }
+        toolbar.css('left', parseInt(offset.left) + width + distance_left + 'px');
+        toolbar.css('top', parseInt(offset.top) + distance_top + 'px');
+        toolbar.show();
+    };
+
+    $('.result-list input[type=text]:not(.no-fillable), .result-list select').focus(function () {
+        toggle_toolbar(this);
+    });
+
+    /*
+     * Mousedown because as we use the "blur" in #float_toolbar
+     * "click" does not work
+     */
+    $('#float_toolbar').mousedown(function (event) {
+        var MAX_ROWS = 100;
+        var input_id = $('#float_toolbar').data('input_id');
+        var input_name = $('#float_toolbar').data('input_name');
+        var matcher = /(.*)-([0-9]+)-(.*)/;
+        var results, pre, number, post, el, sourceField;
+        if (input_id !== '') {
+            results = matcher.exec(input_id);
+            sourceField = $('#' + input_id);
+        } else if (input_name !== '') {
+            results = matcher.exec(input_name);
+            sourceField = $('input[name="' + input_name + '"]');
+        }
+        pre = results[1];
+        number = results[2];
+        post = results[3];
+        for (var i = 0; i < MAX_ROWS; i++) {
```

Comment:
try to use some regex to get all inputs and operate on returned jquery "list" without loop